bfh/fields.py

```diff
@@ -33,6 +33,10 @@ class so you can implement your own if you want.
 ]
 
 
+def get_default(value):
+    return value() if callable(value) else value
```

Comment:
Er, now i see this a function, not a method. I'd suggest just moving the evaluation the getter method.